src/main/java/echoServer/HTTP/HTTPResponse.java

```diff
@@ -0,0 +1,24 @@
+package echoServer.HTTP;
+
+import static echoServer.HTTP.StatusCode.PAGE_NOT_FOUND;
+
+public class HTTPResponse {
+
+    // Int code = new Int();
+    // use enum with values for status code type
+    StatusCode statusCode;
+
+    public HTTPResponse(StatusCode statusCode) {
+        this.statusCode = statusCode;
+    }
+
+
+    public static String buildResponse(StatusCode statusCode){
```

Comment:
Conventionally, it would probably make more sense to call this method `toString`. `build` implies a Builder pattern which you aren't using.